.github/CODEOWNERS

```diff
@@ -0,0 +1,10 @@
+* @cyberark/community-and-integrations-team @conjurinc/community-and-integrations-team @conjurdemos/community-and-integrations-team
```

Comment:
@Tovli - does your team have an alias that should go here instead of the C&I team? Ig not, we should probably remove this line.